Screen.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Screen.h"
+
+LiquidCrystal_I2C lcd(0x27,20,4);
+
+Screen::Screen(){}
+
+void Screen::calibrationAll() {
+  bno.begin();
+  bno.setExtCrystalUse(true);
+  lcd.init();
+  lcd.backlight();
+  writeLCDdown("LCD ready");
+  lcd.clear();
+  lcd.setCursor(0,0);
+  
+  while(bno_.orientationStatus() != 3) {
+    lcd.setCursor(6,0);
+    lcd.print("I´m not ready");
+  }
+  lcd.setCursor(6,1);
+  lcd.print("I´m ready");
+
+  delay(kDelayAfterBNO);
+
+  pinMode(LED, INPUT); 
+  digitalWrite(LED, LOW);
+  
+  pinMode(motors.kMotorLeftForward1, OUTPUT);
+  pinMode(motors.kMotorLeftForward2, OUTPUT);
+  pinMode(motors.kMotorLeftBack1, OUTPUT);
+  pinMode(motors.kMotorLeftBack2, OUTPUT);
+  pinMode(motors.kMotorRightForward1, OUTPUT);
+  pinMode(motors.kMotorRightForward2, OUTPUT);
+  pinMode(motors.kMotorRightBack1, OUTPUT);
+  pinMode(motors.kMotorRightBack2, OUTPUT);
+
+  sensors_event_t event;
+  bno.getEvent(&event);
+}
+
+void Screen::writeNumLCD(const int num) {
+  lcd.clear();
+  lcd.print(num);
+}
+
+void Screen::writeLyricsLCD(const char letra) {
+  lcd.print(letra);
+}
+
+void Screen::writeLCD(const String sE1, const String sE2) {
+  lcd.clear();
+  lcd.print(sE1);
+  lcd.setCursor(0, 1);
+  lcd.print(sE2);
+}
+
+void Screen::writeLCDdown(const String sE1) {
+  lcd.setCursor(0, 1);
+  lcd.print(sE1);
+}
```

Comment:
Decide for a format, what I mean is, if I call writeNumLCD() it clears the screen and prints the num, whereas writeLyricsLCD only prints the letter, and so on. How are we supposed to know how they'll print without having to enter to the cpp file to read the code?